src/views/product-downloads-view/helpers.ts

```diff
@@ -11,6 +13,9 @@ const PLATFORM_MAP = {
   Linux: 'linux',
 }
 
+// exclude pre-releases and such
+const VALID_SEMVER_REGEX = /^\d+\.\d+\.\d+$/
```

Comment:
Super minor & non-blocking nit, but I wonder whether it'd be worth the reassurance of using [semver](https://www.npmjs.com/package/semver). (not gonna lie i find the docs to be quite hard to parse, but I think it'd be something like `semver.valid(maybeSemverString)` to check it's valid and then `semver.satisfies(maybeSemverString, '*')` to check that it's not a pre-release).

Reply:
Woah this is a great idea to look into! We're using the semver package for sorting already, so I'm guessing one of these validation functions wouldn't be hard to add. I'll work on adding this in one of my follow-up PRs!